### ISSUE TITLE
Detect register format error

### ISSUE THREAD

**Issue by treelite** [systemjs/systemjs#1071]
`leadingCommentAndMetaRegEx` can't filter multi comments, for example:

```js
/**
 * @file file comment
 * ...
 */

/**
 * module comment
 * ...
 */
System.register(['xxxx'], function () {}); 
```

In this case, `detectRegisterFormat()` will return `false` unfortunately

Maybe the `leadingCommentAndMetaRegEx` need to handle the empty lines between each comment ?

**Comment by guybedford** [comment]
Thanks, it's supposed to handle these cases, but may be something more subtle in the regex.

**Comment by guybedford** [comment]
Released in 0.19.19.
